src/binaryen-c.h

```diff
@@ -659,6 +659,15 @@ int BinaryenModuleValidate(BinaryenModuleRef module);
 // Runs the standard optimization passes on the module.
 void BinaryenModuleOptimize(BinaryenModuleRef module);
 
+// Sets the optimization level to use. 0, 1, 2 correspond to -O0, -O1, -O2, etc.
+void BinaryenSetOptimizeLevel(int level);
```

Comment:
Alternatively to this API, another option is to just expect people to use the existing RunPasses on inputs like `-O` etc.  - same as on the console? That could trivially reuse the code from src/tools/optimization-options.h.

Or, there could be an Optimize() variant with more params.

Thoughts?

Reply:
RunPasses has the disadvantage that one has to know exactly which passes are available in the current version of the API. It also seemed to me that some passes do different things depending on the actual levels?

Reply:
Yeah, good point about passes depending on the optimize levels too. So I think your API choices in this PR make sense.